backend/app/cmd/server.go

```diff
@@ -321,7 +321,9 @@ func (s *ServerCommand) Execute(_ []string) error {
 	return nil
 }
 
-// HandleDeprecatedFlags sets new flags from deprecated returns their list
+// HandleDeprecatedFlags sets new flags from deprecated returns their list.
+// Returned list has DeprecatedFlag.Old and DeprecatedFlag.Version set, and DeprecatedFlag.New is optional
+// (as some entries are removed without substitute).
 func (s *ServerCommand) HandleDeprecatedFlags() (result []DeprecatedFlag) {
```

Comment:
maybe we don't need two methods to handle deprecated flags and should combine them?

Reply:
I thought of combining them and differentiating the results by flag, but found out that the complexity of resulting function is too high and there really should be two functions: one reading old flags and setting new ones if old ones are still in use, and other just detecting collisions but sets nothing.

Reply:
well, i have nothing against having two functions inside the command to address the complexity, it makes sense. However I don't really see the reason the have two "entry points", i.e. interface functions doing a very similar thing. Maybe keep two and call conflict thingy from the handler without extending the interface?